mods/ra/maps/soviet-05/soviet05.lua

```diff
@@ -144,7 +144,10 @@ end
 Tick = function()
 	if Greece.HasNoRequiredUnits() and GoodGuy.HasNoRequiredUnits() then
 		player.MarkCompletedObjective(KillAll)
-		player.MarkCompletedObjective(HoldObjective)
+
+		if HoldObjective then
```

Comment:
Since I'm not so good with our lua scripts and couldn't find the info in our code:
How exactly does this check work? Do objectives return a boolean value, or does this just check if it's not null?

Reply:
Check if it the variable is not null (or true, but it won't be true).

Reply:
https://logs.openra.net/index.php?year=2019&month=02&day=23#22:09:59